pvlib/pvsystem.py

```diff
@@ -1778,9 +1791,48 @@ def singlediode(photocurrent, saturation_current, resistance_series,
 
     Notes
     -----
-    The solution employed to solve the implicit diode equation utilizes
-    the Lambert W function to obtain an explicit function of V=f(i) and
-    I=f(V) as shown in [2].
+    The default method employed is an explicit solution using [4] to find an
```

Comment:
More editing: "The default method is an explicit solution from [4] that finds "

Reply:
✔️